obp-api/src/main/scala/code/bankconnectors/KafkaMappedConnector.scala

```diff
@@ -744,7 +745,7 @@ object KafkaMappedConnector extends Connector with KafkaHelper with MdcLoggable
     val viewsDeleted = Views.views.vend.removeAllViews(bankId, accountId)
 
     //delete account
-    val account = getBankAccount(bankId, accountId)
+    val account = getBankAccountOld(bankId, accountId)
```

Comment:
Connector methods should not have overloaded methods, it cause starConnector can't work, so here rename one old method to getBankAccountOld.